draft-ferrieuxhamchaoui-quic-lossbits.md

```diff
@@ -287,23 +286,23 @@ The use of the loss bits is negotiated using a transport parameter:
 
 loss_bits (0x1055):
 
-: The loss bits transport parameter is an integer value that can be set
-  to 0 or 1 indicating the level of QL bits support.
+: The loss bits transport parameter is an integer value, encoded as a
+  variable-length integer, that can be set to 0 or 1 indicating the level of
+  QL-bits support.
 
-When loss_bits is set to 0, the peer is allowed to change R-bits in the
+When loss_bits parameter is present, the peer is allowed to change R-bits in the
 short packet header to QL-bits if the peer sends loss_bits=1.
 
-When loss_bits is set to 1, the sender will change R-bits to QL-bits if
-the peer sends the loss_bits transport parameter set to either 0 or 1.
+When loss_bits is set to 1, the sender will change R-bits to QL-bits if the peer
+includes the loss_bits transport parameter.
 
 A client MUST NOT use remembered value of loss_bits for 0-RTT connection.
 
-# Mechanics
```

Comment:
I prefer grouping the QUIC changes introduced by the extension into a single section.  Perhaps "mechanics" is not the right title -- is that why you propose dropping it?

Reply:
Yes.  I did not like "Mechanics" as a section.  If you can come up with a better title and have an intro sentence, make a PR.

Also, would you include "Transport Parameter" section as a subsection to that one?

Reply:
OK, I'll try to come up with something.

Good point about "Transport Parameter" section.